### PR TITLE
Added Flutter course link by Codepur

### PR DESCRIPTION
**Pull request by tejas786u**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Add resource

### Description
Very good free flutter course especially for beginners

### Why is this valuable (or not)?
This is a single video link that teaches from scratch for any beginners who want to learn flutter

### How do we know it's really free?
It's the YouTube video link

## Checklist:
- [-] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [-] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [-] Include author(s) and platform where appropriate.
- [-] Put lists in alphabetical order, correct spacing.
- [-] Add needed indications (PDF, access notes, under construction).
- [-] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -272,6 +272,7 @@
 * [Flutter Series 2020](https://www.youtube.com/playlist?list=PLDzeHZWIZsTo3Cs115GXkot28i406511Y) - CodeHelp - by Babbar
 * [Flutter Tutorial For Beginners in Hindi](https://youtube.com/playlist?list=PLMkkZSS5OjPIwDyHHKVex6zr008U1-sWM) - Geeks Rank
 * [Flutter Widgets in Hindi](https://www.youtube.com/playlist?list=PLz7ymP4HzwSH3vAnhDWLkO2TLbwGeigl7) - Ahirlog
+* [Master Flutter in Just 8 Hours | Full Course Hindi](https://www.youtube.com/watch?v=j-LOab_PzzU) - Codepur
```
Comment (sohansai) [inline courses/free-courses-hi.md:275]:
```suggestion
* [Master Flutter in Just 8 Hours - Full Course Hindi](https://www.youtube.com/watch?v=j-LOab_PzzU) - Codepur
```

Comment (Thenlie) [reply courses/free-courses-hi.md:275]: For this we can escape the pipe with `\|`

Comment (Thenlie) [inline courses/free-courses-hi.md:275]: For youtube, we prefer "playlist" links and the name of the producer/creator. View [these docs](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#platforms-and-access-notes) for more information. Once you have updated you PR, you can add “changes made” to the title and reviewer will take another look as soon as they are available. Thanks!

**Comment by eshellman** [comment]
duplicate

**Pull request closed without merging**